src/bento/components/bento-date-countdown/1.0/bento-date-countdown.css

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
fmi - why does this file need to exist? I think currently we build the bento css by building the amp css, copying it, and changing the name of any element selectors from `amp-foo` -> `bento-foo`. 

same for date-display.css and embedly-card.css

Reply:
oh hmm wait i think we'd use the bento css in the bento specific builds because of the [glob patterns we use](https://github.com/ampproject/amphtml/blob/487f3c6d7d8de3054b1d1228335f8c72bd618948/build-system/tasks/extension-helpers.js#L567), so nvm. 

But, we're kind of duplicating CSS between the amp and bento components though, which might be a footgun.. probably not a big deal since we're not making that many CSS changes to those components right now anyway